gym/lib/gym/error_handler.rb

```diff
@@ -104,6 +104,10 @@ def handle_empty_archive
         print "Also, make sure to have a valid code signing identity and provisioning profile installed"
         print "Follow this guide to setup code signing https://docs.fastlane.tools/codesigning/GettingStarted/"
         print "If your intention was only to export an ipa be sure to provide a valid archive at the archive path."
+        print "This error might also happen if your workspace/project file is not in the root directory of your project"
+        print "To workaround that issue, you can add `Dir.chdir('../path/to/dir/containing/proj') do`"
```

Comment:
I think I could fake a commit and push it up which would just be the `dir.chdir` + comments